inc/class-formatter.php

```diff
@@ -42,7 +42,7 @@ public function __construct( &$assoc_args, $fields = null, $prefix = false ) {
 	 *
 	 * @param array $items
 	 */
-	public function display_items( $items, $include_total = true, $order, $orderby ) {
+	public function display_items( $items, $include_total, $order, $orderby ) {
 		if ( 'table' === $this->args['format'] && empty( $this->args['field'] ) ) {
```

Comment:
We will nevertheless need to retain the default value behavior for back compact:
```suggestion
		if ( null === $include_total ) {
			$include_total = true;
		}

		if ( 'table' === $this->args['format'] && empty( $this->args['field'] ) ) {
```

Reply:
Do we need to? That isn't how we fixed the issues in Core, https://core.trac.wordpress.org/ticket/50343#comment:7

A notice would've been issued for 5.x and a fatal in 7.x: https://3v4l.org/kIOX1

Reply:
You're right, that doesn't do much. Also, I just realized that this is not the framework formatter's `display_items()` method, but rather a custom version limited to this package. And it is only ever used will all arguments anyway.